### ISSUE TITLE
Asking about the choosing  anchor ratios

### ISSUE THREAD

**Issue by hunglv213** [AIZOOTech/FaceMaskDetection#53]
Hi, thank you for your great work! 

I would you to ask you about the anchor ratios. 

> anchor_ratios = [[1, 0.62, 0.42]] * 5

If I change the order of anchor_ratios, can it hurt the performance? For example, 

> anchor_ratios = [[0.62, 0.42,1]] * 5

Thank you in advance.

**Comment by AIZOOTech** [comment]
@hunglv213 Yes, you can not change the order. What's your reason to change the order?

**Comment by hunglv213** [comment]
I got it. Therefore, these scales are set default by order. 

However, I am curious about how to set the scales properly. 

Is that the first one always go with 1? 

I've tried to use K-mean to analyze the bounding box distribution. Found out the most suitable scale for width and height. However, I don't know how to put it into the the order.

Thank you again.
